### ISSUE TITLE
build workshop 3 - Testing with an existing database, include continuous integration setup

### ISSUE THREAD

**Issue by PiotrBerebecki** [foundersandcoders/master-reference#467]
Link to workshop: https://github.com/Akin909/express-and-testing-workshop

**Comment by Akin909** [comment]
@jsms @des-des @PiotrBerebecki Having attempted to write this workshop with shot and tape ive found that there is little to no mention of using that combination of tools to test express and If an advantage of teaching express is giving students access to the resources that come with express then it would be advisable to use testing frameworks which are more commonly used with express such as mocha and chai or at the very least tape with supertest as shot is designed to work with hapi, hopefully this is acceptable and if not I'd appreciate some pointer as to any other suggestions.

**Comment by PiotrBerebecki** [comment]
Hi @Akin909, most of the resources I've found were also referring to Mocha and Chai. The advise is that we should use tape and supertest. Have a look here: https://github.com/foundersandcoders/master-reference/pull/490

I haven't used this combination yet so we we will indeed need some support from the FAC community.

**Comment by des-des** [comment]
@akin909 yeah, I have had some of my projects with tape / supertest. It seems like a fine combo although I have not looked into it.

I think we should stick with tape, since that is what the students have been using through out the course. A bit hard to follow your comment maybe add some full stops.

But yeah any specific questions I would be happy to anwser

https://github.com/foundersandcoders/open-tourism-platform/blob/master/tests/controllers/event.test.js
